src/pykiso/test_coordinator/test_execution.py

```diff
@@ -474,12 +475,19 @@ def execute(
 
         log_file_path = get_logging_options().log_path
         # TestRunner selection: generate or not a junit report. Start the tests and publish the results
+        # pykiso --junit
         if report_type == "junit":
-            junit_report_name = time.strftime(f"%Y-%m-%d_%H-%M-%S-{report_name}.xml")
-            project_folder = Path.cwd()
-            reports_path = project_folder / "reports"
-            junit_report_path = reports_path / junit_report_name
-            reports_path.mkdir(exist_ok=True)
+            report_path = junit_path
```

Comment:
do we need the renaming ?
Just proceed with junit_path instead of report_path

Reply:
ok, i will rename it